gen/definitions/tag.yaml

```diff
@@ -0,0 +1,71 @@
+---
+name: Tag
+rest_endpoint: /dna/intent/api/v1/tag
+id_from_query_path: response.0
+id_from_query_path_attribute: id
+put_id_include_path: id
+import_no_id: true
+data_source_no_id: true
+doc_category: Tags
+attributes:
+  - model_name: name
+    response_data_path: response.0.name
+    type: String
+    mandatory: true
+    query_param: true
+    example: Tag1
+  - model_name: description
+    response_data_path: response.0.description
+    type: String
+    description: Description of the tag
+    example: Tag1 Description
+  - model_name: systemTag
+    response_data_path: response.0.systemTag
+    type: Bool
+    description: true for system created tags, false for user defined tag
+    example: false
+  - model_name: dynamicRules
+    response_data_path: response.0.dynamicRules
+    type: List
+    description: Dynamic rules details
+    exclude_test: true
+    attributes:
+      - model_name: memberType
+        type: String
+        description: memberType of the tag (e.g. networkdevice, interface)
+      - model_name: values
+        tf_name: rule_values
```

Comment:
I think we can remove the `tf_name` here, as the parent element is already named `dynamic_rules`.

Reply:
done